src/Form/DeveloperAttributesSettingsForm.php

```diff
@@ -72,12 +75,26 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     uasort($fields, [FieldConfig::class, 'sort']);
 
     $options = $default_values = [];
+    /** @var \Drupal\apigee_edge\FieldStorageFormatManager $format_manager */
+    $format_manager = \Drupal::service('plugin.manager.apigee_field_storage_format');
     /** @var \Drupal\field\FieldConfigInterface $field */
     foreach ($fields as $field) {
       $options[$field->getName()] = [
-        'field_name' => $field->getLabel(),
-        'attribute_name' => preg_replace('/^field_(.*)$/', '${1}', $field->getName()),
+        'field_label' => $field->getLabel(),
+        'field_name' => $field->getName(),
+        'field_type' => $field->getType(),
+        'attribute_name' => static::getAttributeName($field->getName()),
       ];
+      $formatter = $format_manager->lookupPluginForFieldType($field->getType());
+      if (isset($formatter)) {
+        $rc = new \ReflectionClass($format_manager->lookupPluginForFieldType($field->getType()));
+        $short_name = $rc->getShortName();
+        $options[$field->getName()]['field_storage_formatter'] = $short_name;
+      }
+      else {
+        $options[$field->getName()]['field_storage_formatter'] = '-';
```

Comment:
Meh, there is no "#empty-option" for table select, isn't it? What happens if you do not set this array key? Maybe it should be `t('- None -')` if we have to set this key's value.